.gitattributes

```diff
@@ -2,5 +2,3 @@
 *.csv filter=lfs diff=lfs merge=lfs -text
 *.vasp filter=lfs diff=lfs merge=lfs -text
 *.poscar filter=lfs diff=lfs merge=lfs -text
-examples/assets/* filter=lfs diff=lfs merge=lfs -text
-"examples/assets/mp-978534.poscar" filter=lfs diff=lfs merge=lfs -text
```

Comment:
What happened here??

Reply:
The assets folder readme does not display to the user in Git if it is an LFS pointer. The only files in that directory are csv and poscar files, anyway.